src/include/override/sys/generate-syscall.py

```diff
@@ -39,6 +39,10 @@ def parse_syscall_tables(filenames):
  *
  * Use 'ninja -C build update-syscall-tables' to download new syscall tables,
  * and 'ninja -C build update-syscall-header' to regenerate this file.
+ * To add a new architecture, manually download the appropriate table from
+ * https://raw.githubusercontent.com/hrw/syscalls-table/master/data/tables/ to
+ * src/include/override/sys/, then list it in src/include/override/sys/generate-syscall.py
+ * and src/include/override/sys/meson.build and then finally run the above commands.
```

Comment:
Two `and` in one sentence sounds slightly strange. Maybe better to drop the second one and insert a comma before `then`?